src/configs/env.ts

```diff
@@ -313,7 +313,7 @@ export const PRACTITIONER_FORM_NAMESPACE =
   process.env.REACT_APP_PRACTITIONER_FORM_NAMESPACE || '78295ac0-df73-11e9-a54b-dbf5e5b2d668';
 export type PRACTITIONER_FORM_NAMESPACE = typeof PRACTITIONER_FORM_NAMESPACE;
 
-const usersRequestPageSize = process.env.REACT_APP_USERS_REQUEST_PAGE_SIZE || '100';
+const usersRequestPageSize = process.env.REACT_APP_USERS_REQUEST_PAGE_SIZE || '1000';
```

Comment:
Could you add a test that shows this value being used?

Reply:
addressed via https://github.com/onaio/reveal-frontend/pull/1166/commits/a27c53cb68556f68dd9da405285e6eaf0f121296